FrontEnd/src/components/ProfileDetail/DetailedInfo/DataContacts.jsx

```diff
@@ -1,43 +1,33 @@
-import React, { useMemo } from 'react';
+import React from 'react';
 import { PropTypes } from 'prop-types';
 import PhoneEmail from './PhoneEmail';
-import { ReactComponent as FacebookSvg } from './SvgFiles/facebook.svg';
-import { ReactComponent as InstagramSvg } from './SvgFiles/instagram.svg';
 import classes from './DataContacts.module.css';
 
-function DataContacts ({ isAuthorized, data }) {
-    const profile = useMemo(() => {
-        return {
-          id: data.id,
-          edrpou: data.edrpou,
-          founded: data.founded,
-          address: data.address,
-        };
-      }, [data]);
+function DataContacts ({ data }) {
 
     // TODO: implement logic for getting data from db when it's added on server side
 
     const companyData = {
-        'ЄДРПОУ': profile.edrpou,
-        'Рік заснування': profile.founded,
+        'ЄДРПОУ': data.edrpou,
+        'Рік заснування': data.founded,
         'Розмір компанії': '',
         'Аудит': ''
     };
 
     const companyContacts = {
         'Сайт': '',
-        'PhoneEmail': <PhoneEmail isAuthorized={isAuthorized} profileId={data.id}/>,
-        'Адрес(и)': profile.address,
+        'PhoneEmail': <PhoneEmail profileId={data.id} personId={data.person}/>,
+        'Адрес(и)': data.address,
         'Соціальні мережі': [
             {
                 name: 'facebook',
                 url: '',
-                svgPath: <FacebookSvg />,
+                svgPath: <img src={`${process.env.PUBLIC_URL}/svg/facebook.svg`} />,
```

Comment:
There are pros and cons for both options using SVG. Did you verify that we have `PUBLIC_URL` on devci? Does it work in CRA w/o `REACT_APP` prefix?

Reply:
Dev team still has no information about it. PUBLIC_URL is used not only in this component. We'll made changes if needed.